cmd/jaeger/internal/integration/storagecleaner/extension_test.go

```diff
@@ -90,9 +91,12 @@ func TestStorageCleanerExtension(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
+			port := getFreePort()
+			portStr := fmt.Sprintf("%d", port)
```

Comment:
Move string formatting into getFreePort(). You can use a helper from ports/ pkg.